eng/Versions.props

```diff
@@ -26,6 +26,7 @@
     <!-- CodeStyleAnalyzerVersion should we updated together with version of dotnet-format in dotnet-tools.json -->
     <CodeStyleAnalyzerVersion>4.2.0-2.final</CodeStyleAnalyzerVersion>
     <VisualStudioEditorPackagesVersion>17.2.3194</VisualStudioEditorPackagesVersion>
+    <VisualStudioMacEditorPackagesVersion>17.3.67-preview-ga1f07c2812</VisualStudioMacEditorPackagesVersion>
```

Comment:
We can change this to `$(VisualStudioEditorPackagesVersion)` after my editor changes have merged in VS and Roslyn is able to take those newer packages and dependencies.